kong/conf_loader/init.lua

```diff
@@ -660,6 +660,8 @@ local CONF_INFERENCES = {
 
   lmdb_environment_path = { typ = "string" },
   lmdb_map_size = { typ = "string" },
+
+  events_mechanism = { typ = "string" },
```

Comment:
```suggestion
  legacy_worker_events = { typ = "boolean" },
```

May be a better name. Then it can default to `off`.